sdk/src/extras/disk.rs

```diff
@@ -415,18 +415,25 @@ pub mod types {
                 drop(tx);
```

Comment:
up at 399 should we be generating an error in that case? If I understand correctly, the call to `send` is only going to fail when the sending task encounters some failure... in which case that task is already going to report a failure. That is to say: should we just ignore send errors?

Here's the docs for send:

> If the receive half of the channel is closed, either due to [close](https://docs.rs/tokio/latest/tokio/sync/mpsc/struct.Receiver.html#method.close) being called or the [Receiver](https://docs.rs/tokio/latest/tokio/sync/mpsc/struct.Receiver.html) handle dropping, the function returns an error. The error includes the value passed to send.

Reply:
Yes, good point. We should still `break` on a `send` error since the overall request will fail, but should not treat it as an error for the file reader task.